### ISSUE TITLE
Testing Mines: can establish mine fields without HR (for the engineers) 

### ISSUE THREAD

**Issue by Warhammer47** [LordGolias/antistasi#105]
After establishing the minefield and having no HR (HR: 0), the engineers are recruited anyway and the task created.

**P.S. Tested with ACE in the fix_Minefields branch in MP**

**Comment by LordGolias** [comment]
Cannot reproduce. Let me describe the steps so you can identify where the error is:

1. press to create a AT minefield
2. select minefield and mines positions
3. confirm selection
4. money and hr is removed
5. mines are removed from arsenal
6. mission is created
7. mission is initialized (task created)

What I tried to reproduce:

1. new game
2. removed all HR from FIA (`[-10,0] call AS_fnc_changeFIAmoney` on server)
3. pressed to create a minefield => rejected: no mines
4. added mines to box
5. pressed to create a minefield => rejected, no HR or money
6. added HR back to FIA (`[10,0] call AS_fnc_changeFIAmoney` on server)
7. pressed to create a minefield => mission created

In other words, the HR is removed once the mission is created, and it will remain allocated for the mission.

**Comment by LordGolias** [comment]
Re-open if you do not reproduce or agree with the above :P
